app/frontend/src/components/AdminTabs.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Tabs, Tab } from 'react-bootstrap';
+import Users from './Users';
+import Roles from './Roles';
+import Activities from './Activities';
+
+class AdminTabs extends React.Component {
+    constructor(props, context) {
+      super(props, context);
+  
+      this.handleSelect = this.handleSelect.bind(this);
+      this.toggleRefreshRoles = this.toggleRefreshRoles.bind(this);
+      this.state = {
+        refreshRoles: false,
+        key: 1
+      };
+    }
+  
+    handleSelect(key) {
+      this.setState({ key });
+    }
+
+    toggleRefreshRoles(boolean) {
+      this.setState({ refreshRoles: boolean });
+    }
+  
+    render() {
+      return (
+        <Tabs
+          activeKey={this.state.key}
+          onSelect={this.handleSelect}
+          id="admin-tabs"
+        >
+          <Tab eventKey={1} title="User Management">
+            <Users toggleRefreshRoles={this.toggleRefreshRoles} refreshRoles={this.state.refreshRoles}/>
+          </Tab>
+          <Tab eventKey={2} title="User Roles">
+            <Roles toggleRefreshRoles={this.toggleRefreshRoles}/>
+          </Tab>
+          <Tab eventKey={3} title="Attendance Activities">
+            <Activities />
+          </Tab>
+          <Tab eventKey={4} title="Student Profile Fields">
+            Tab 4 content
```

Comment:
nitpicky, but this looks like placeholder text - was something supposed to go here for this PR or is that for later?

Reply:
This is for later- this PR deals with attendance activities only, functionality for creating new student profile fields will be a separate thing.